res/values/strings.xml

```diff
@@ -593,40 +592,40 @@
+    <string name="autocrypt_send_asm_explain_before">Message your Autocrypt end-to-end encryption to other Autocrypt-compliant apps you use.\n\nType in the confirmation code displayed here on the other device afterwards.</string>
     <string name="autocrypt_send_asm_button">Send Autocrypt Setup Message</string>
-    <string name="autocrypt_send_asm_explain_after">Your setup has been sent to yourself. Switch to the other device and open the setup message. You should be asked for a setup code. Enter the following digits:\n\n%1$s</string>
+    <string name="autocrypt_send_asm_explain_after">Your Autocrypt end-to-end encryption has been messaged to yourself. Open it on the other device and type in this confirmation code:\n\n%1$s</string>
     <string name="autocrypt_prefer_e2ee">Prefer end-to-end encryption</string>
     <string name="autocrypt_asm_subject">Autocrypt Setup Message</string>
-    <string name="autocrypt_asm_general_body">This is the Autocrypt Setup Message used to transfer your end-to-end setup between clients.\n\nTo decrypt and use your setup, open the message in an Autocrypt-compliant client and enter the setup code presented on the generating device.</string>
-    <string name="autocrypt_asm_click_body">This is the Autocrypt Setup Message used to transfer your end-to-end setup between clients.\n\nTo decrypt and use your setup, tap or click on this message.</string>
+    <string name="autocrypt_asm_general_body">Open this message in an Autocrypt-compliant client and enter the confirmation code from the device it was sent from.</string>
+    <string name="autocrypt_asm_click_body">Tap or click on this message to use this device like the one it was sent from.</string>
     <string name="autocrypt_continue_transfer_title">Autocrypt Setup Message</string>
-    <string name="autocrypt_continue_transfer_please_enter_code">Please enter the setup code displayed on the other device.</string>
-    <string name="autocrypt_continue_transfer_succeeded">End-to-end setup transferred. This device is now ready to use Autocrypt with the same setup as the other device.</string>
+    <string name="autocrypt_continue_transfer_please_enter_code">Please enter the confirmation code displayed on the other device.</string>
+    <string name="autocrypt_continue_transfer_succeeded">Autocrypt End-to-end encryption transferred to yourself. This device is now ready to use like the other device.</string>
     <string name="autocrypt_continue_transfer_retry">Retry</string>
-    <string name="autocrypt_bad_setup_code">Bad setup code. Please try again.\n\nIf you do not remember the setup code, send a new Autocrypt Setup Message from the other device.</string>
+    <string name="autocrypt_bad_setup_code">Wrong confirmation code. Please try again.\n\nIf you do not remember it, send a new Autocrypt Setup Message from the other device.</string>
```

Comment:
one may like or dislike the term "setup code", however, changing it is not only a thing of this file, it is used at several other places in other apps as well.

also, not sure if "confirmation code" is better at all.

Reply:
Be that as it may, the code only belongs to the confirmation-part, not to the setup itself.